src/components/MyProfile/Profile.tsx

```diff
@@ -0,0 +1,142 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { IcGithub, IcRevise } from '../../assets';
+import { handleClickLink } from '../../utils/handleClickLink';
+import ProfileEdilt from './ProfileEdit';
+
+const Profile = () => {
+  const [modalOn, setModalOn] = useState(false);
+  const info = {
+    profileImg: '',
+    language: 'Javascript',
+    githubUrl: '',
+    nickname: '매링구',
+    comment: '안녕하세요 풀스택 개발자 코딩하는 갱얼쥐입니다',
+  };
```

Comment:
P2: 아래처럼 구조분해할당을 해주면 `info.` 없이도 객체를 활용할 수 있어요 !
```suggestion
  const info = {
    profileImg: '',
    language: 'Javascript',
    githubUrl: '',
    nickname: '매링구',
    comment: '안녕하세요 풀스택 개발자 코딩하는 갱얼쥐입니다',
  };
  
  const { profileImg, language, githubUrl, nickname, comment } = info;
```